### DISCUSSION TITLE
Missing Store Folder

### DISCUSSION THREAD

**Discussion by cyriaque-ovaga** [nuxt/nuxt#18346] [Questions] [answered]
Hello, we had the store folder to save our states. How to do it on nuxt3 ?

**Comment by lucpotage** [comment, marked as answer]
You have access to the new `useState` composable. https://v3.nuxtjs.org/docs/usage/state

There is also the `pinia` module but it’s not ready yet: https://github.com/posva/pinia/tree/v2/packages/nuxt

**Comment by pi0** [comment]
Please follow https://github.com/nuxt/framework/discussions/571 for more updates about `store/` dir with vuex integration
